lib/services/network_manager.dart

```diff
@@ -76,9 +94,9 @@ Future<void> _onConnectivityChange(
   _networkAutomationLogger.finest(
       "Network Change: ${connections?.map((element) => element.toString()).join(", ") ?? "None (likely a manual function call)"}");
   connections ??= await Connectivity().checkConnectivity();
-  final [offlineModeActive, baseUrlChanged] = await Future.wait([
+  final [_, baseUrlChanged] = await Future.wait([
```

Comment:
Did the linter yell at you because you're not using the variable? If so, just ignore it. I think it's good to know what a returned value is/represents.

Reply:
Artifact from trying to make offlineModeActive return a void